virttest/utils_v2v.py

```diff
@@ -1179,7 +1179,9 @@ def setup_esx_ssh_key(hostname, username, password, port=22):
             port=port,
             password=password,
             prompt=r"[\#\$\[\]]",
-            preferred_authenticaton='password,keyboard-interactive')
+            verbose=True,
+            preferred_authenticaton='password,keyboard-interactive',
+            user_known_hosts_file='~/.ssh/known_hosts')
```

Comment:
$HOME/.ssh/known_hosts maybe more safer in some situation than '~/.ssh/known_hosts' when assign value to user_known_hosts_file

Reply:
Done